webrender/src/resource_cache.rs

```diff
@@ -62,28 +62,6 @@ pub struct CacheItem {
     pub uv1: DevicePoint,
 }
 
-#[derive(Clone, Hash, PartialEq, Eq, Debug, Ord, PartialOrd)]
```

Comment:
Does this need to be removed? The idea here is that when passing around the glyph keys, they can be smaller by not including duplicated fields that are the same for an entire text run (the glyph options, and render mode).